common/plugins/com.liferay.ide.core/plugin.xml

```diff
@@ -71,7 +71,16 @@
          point="org.eclipse.core.resources.natures">
       <runtime>
          <run
-               class="com.liferay.ide.core.LiferayNature">
+               class="com.liferay.ide.core.LiferayProjectNature">
+         </run>
+      </runtime>
+   </extension>
+   <extension
+         id="com.liferay.ide.core.liferayWorkspaceNature"
+         point="org.eclipse.core.resources.natures">
+      <runtime>
+         <run
+               class="com.liferay.ide.core.LiferayWorkspaceNature">
```

Comment:
Are you thinking that we need a separate nature?  I haven't thought about it much, do we really need a 2nd nature?  If so, I'm fine with it, but I don't want to create a new nature unless we for sure need it.